packages/components/src/input-control/backdrop.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+import { memo } from '@wordpress/element';
+/**
+ * Internal dependencies
+ */
+import { Fieldset, Legend, LegendText } from './styles/input-control-styles';
+
+function Backdrop( {
+	disabled = false,
+	isFloating = false,
+	isFloatingLabel = false,
```

Comment:
What's the difference between these two?

Reply:
One tracks the setting preference (and validity) of it via the `isFloatingLabel` prop. The other tracks the floating state.

The varying combinations adjust the styles for the HTML elements

Reply:
Perhaps `isFloatingLabel` should be renamed to `useFloatingLabel` or `floatingLabel` or something like that?

Reply:
I don't think we can use the `use` prefix, since that's associated with hooks. I think ESLint might actually complain.

Prefixing with `is` typically indicates a `boolean` value, which this one is.

Reply:
How about `isFloatingLabelStyle` or `isFloatingLabelType`?

Reply:
Hmm.. I'm finding those names trickier to understand. I'm not sure what the best names may be.

At the moment, both props are shared throughout various sub-components of `InputControl` for both functionality and aesthetics. Because of that, they are consistent within `InputControl`. Unless the current names are overly confusing or broken, I think it may be simpler to keep them.